src/pkcs11/pkcs11-session.c

```diff
@@ -251,6 +251,7 @@ CK_RV C_GetSessionInfo(CK_SESSION_HANDLE hSession,	/* the session's handle */
 	struct sc_pkcs11_session *session;
 	struct sc_pkcs11_slot *slot;
 	int logged_out;
+	const char* name;
```

Comment:
we use the following style in opensc:
```suggestion
	const char *name;
```
The same pattern appears on couple of more places in this PR.